belgif-rest-problem-it/belgif-rest-problem-it-common/src/main/java/io/github/belgif/rest/problem/AbstractOpenApiValidationSpringBootIT.java

```diff
@@ -0,0 +1,172 @@
+package io.github.belgif.rest.problem;
+
+import static org.hamcrest.Matchers.*;
+
+import org.junit.jupiter.api.Test;
+
+import io.restassured.specification.RequestSpecification;
+
+abstract class AbstractOpenApiValidationSpringBootIT {
+
+    protected abstract RequestSpecification getSpec();
+
+    @Test
+    void validPathParamWorksTest() {
+        getSpec().when().get("/myFirstPath/abc1234567").then().assertThat()
+                .statusCode(200).log().all()
+                .body(equalTo("All good!"));
+    }
+
+    @Test
+    void unknownQueryParamTest() {
+        getSpec().when().get("/myFirstPath?myUnknownParam=123").then().assertThat()
+                .statusCode(400).log().all()
+                .body("type", equalTo("urn:problem-type:belgif:badRequest"))
+                .body("issues[0].type", equalTo("urn:problem-type:belgif:input-validation:unknownInput"))
+                .body("issues[0].title", equalTo("Unknown query parameter."))
+                .body("issues[0].in", equalTo("query"))
+                .body("issues[0].name", equalTo("myUnknownParam"))
+                .body("issues[0].value", equalTo("123"))
+                .body("issues[0].detail",
+                        equalTo("Query parameter 'myUnknownParam' is unexpected on path \"/myFirstPath\""));
```

Comment:
For all detail and title fields: avoid testing on an exact value within a free-text description.
It makes the tests fragile: these descriptions might be changed freely, especially when created by a 3rd party library. It's not really a regression/bug when the wording of the description changes.

Checking that the string is non-empty may suffice.